docs/source/install.rst

```diff
@@ -85,3 +80,9 @@ will allow you to change the code without reinstalling Fab every time::
 Please be aware of some considerations when
 `using pip and conda <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#using-pip-in-an-environment>`_
 together.
+
+Containers
+----------
+You can create a development environment for running Fab using the docker file in Fab's github repo.
+For example, PyCharm can use the interpreter inside the container and will automatically volume mount and PYTHONPATH
+Fab.
```

Comment:
This is good but for production release and further beta releases we will need to be able to install the tool without using Conda. e.g. into a python venv.

What is the state of this?

Reply:
We haven't had to use a venv for Fab yet but this is certainly something useful to do soon. https://github.com/metomi/fab/projects/8#card-86192865